frontend/public/components/monitoring/alert-manager-receiver-forms.tsx

```diff
@@ -403,7 +408,7 @@ const ReceiverBaseForm: React.FC<ReceiverBaseFormProps> = ({ obj, titleVerb, sav
           <RoutingLabels
             routeLabels={routeLabels}
             setRouteLabels={setRouteLabels}
-            saveAsDefaultReceiver={saveAsDefaultReceiver}
+            showAddLabel={showAddLabel}
```

Comment:
It seems like this could maybe be `showAddLabel={isDefaultReceiver}`, which would remove the need for `showAddLabel` and simplify things a bit, but is that not the case?